test/parallel/test-cluster-worker-forced-exit.js

```diff
@@ -49,17 +47,18 @@ checkUnforced();
 checkForced();
 
 function checkUnforced() {
-  cluster.fork()
-    .on('online', function() { this.disconnect(); })
-    .on('exit', common.mustCall(function(status) {
-      assert.strictEqual(status, SENTINEL);
-    }));
+  const worker = cluster.fork();
+  worker
+    .on('online', common.mustCall(() => worker.disconnect()))
+    .on(
+      'exit',
```

Comment:
I suggest breaking the lines a bit differently, see e.g. https://github.com/nodejs/node/blob/8b4af64f50c5e41ce0155716f294c24ccdecad03/test/parallel/test-http2-ping-unsolicited-ack.js#L19-L25